src/org/thoughtcrime/securesms/loki/redesign/messaging/LokiPublicChatPoller.kt

```diff
@@ -260,26 +263,30 @@ class LokiPublicChatPoller(private val context: Context, private val group: Loki
         if (displayNameUpdatees.isEmpty()) { return }
         val hexEncodedPublicKeys = displayNameUpdatees
         displayNameUpdatees = setOf()
-        api.getDisplayNames(hexEncodedPublicKeys, group.server).successBackground { mapping ->
-            for (pair in mapping.entries) {
-                val senderDisplayName = "${pair.value} (...${pair.key.takeLast(8)})"
-                DatabaseFactory.getLokiUserDatabase(context).setServerDisplayName(group.id, pair.key, senderDisplayName)
-            }
+        api.getDisplayNames(hexEncodedPublicKeys, group.server).success { mapping ->
```

Comment:
Same thing here, `successBackground` just does whatever you did

Reply:
Like I said I feel it's a lot more obvious this way

Reply:
We can chat about this and potentially move back to the `successBackground` thing if you feel strongly about it but I'm merging this first to get rid of long polling